FFMpegCore/FFProbe/FFProbe.cs

```diff
@@ -117,7 +118,12 @@ private static Instance PrepareInstance(string filePath, int outputCapacity, FFO
             FFProbeHelper.RootExceptionCheck();
             FFProbeHelper.VerifyFFProbeExists(ffOptions);
             var arguments = $"-loglevel error -print_format json -show_format -sexagesimal -show_streams \"{filePath}\"";
-            var instance = new Instance(GlobalFFOptions.GetFFProbeBinaryPath(), arguments) {DataBufferCapacity = outputCapacity};
+            var instance = new Instance(new ProcessStartInfo(GlobalFFOptions.GetFFProbeBinaryPath(), arguments)
```

Comment:
This is a bit messy to read. Could you introduce a local variable for the ProcessStartInfo?
Otherwise looks good 👍

Reply:
Okay, I will create a new PR 😀